st2tests/st2tests/config.py

```diff
@@ -178,82 +187,103 @@ def _register_auth_opts():
         cfg.IntOpt('token_ttl', default=86400, help='Access token ttl in seconds.'),
         cfg.BoolOpt('debug', default=True)
     ]
+
     _register_opts(auth_opts, group='auth')
 
 
 def _register_action_sensor_opts():
     action_sensor_opts = [
-        cfg.BoolOpt('enable', default=True,
-                    help='Whether to enable or disable the ability ' +
-                         'to post a trigger on action.'),
-        cfg.StrOpt('triggers_base_url', default='http://127.0.0.1:9101/v1/triggertypes/',
-                   help='URL for action sensor to post TriggerType.'),
-        cfg.IntOpt('request_timeout', default=1,
-                   help='Timeout value of all httprequests made by action sensor.'),
-        cfg.IntOpt('max_attempts', default=10,
-                   help='No. of times to retry registration.'),
-        cfg.IntOpt('retry_wait', default=1,
-                   help='Amount of time to wait prior to retrying a request.')
+        cfg.BoolOpt(
+            'enable', default=True,
+            help='Whether to enable or disable the ability to post a trigger on action.'),
+        cfg.StrOpt(
+            'triggers_base_url', default='http://127.0.0.1:9101/v1/triggertypes/',
+            help='URL for action sensor to post TriggerType.'),
+        cfg.IntOpt(
+            'request_timeout', default=1,
+            help='Timeout value of all httprequests made by action sensor.'),
```

Comment:
Needs units specified. I assume minutes since that usually the HTTP timeout value, but in a low enough latency environment, one second might also be a decent default.

Nitpick: Should be "HTTP requests".